cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     resources:
       limits:
         cpu: 100m
+        memory: 200Mi
```

Comment:
no, Saad's report in that issue is only for a couple of hours. The limit I chose here is based on over 2 days. Both of us are running the soaking test against our default configurations.